src/custom/utils/metadata.ts

```diff
@@ -0,0 +1,51 @@
+export enum MetadataKind {
```

Comment:
Probably in the future is good idea to move certain things to a SDK. We received that feedback from balancer, that in CowSwap code there's logics that would be useful for them. 

This metadata is something that could be moved to the SDK id we do it.

Reply:
@anxolin the Metadata types and functions were merged into develop in https://github.com/gnosis/cowswap/commit/f7abdd92573d685a913d0b7bd91cbf5e6b7f5b85

I'll rebase this PR later to clean up those commits, but will definitely take into account your comments.